Cargo.toml

```diff
@@ -34,6 +34,7 @@ include = [
 
 [package.metadata.docs.rs]
 features = ["doc"]
+cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples=examples"]
```

Comment:
Add tutorial folder here too?

Reply:
Only `all` and `examples` are supported arguments.